parm/ccpp_control.nml.IN

```diff
@@ -206,6 +206,16 @@
        nspinup        = 1000
        iseed_ca       = 0
        isppt_deep     = .false.
+       do_RRTMGP      = @[DO_RRTMGP]
```

Comment:
Just asking, but is ccpp_control the best namelist to put this in? Should there be a separate one for RRTMGP? I see that DO_RRTMGP is set externally such that the other RRTMGP variables become moot if it is set to false.

Reply:
We want to avoid a proliferation of namelist templates in parm/ - something that EMC is adamant about for good reasons. Therefore, this makes a lot of sense. Is also done for NoahMP.

Reply:
Sounds good. This seems to mark a change from past behavior.

Reply:
Actually, not. We were asked to combine those as much as it makes sense more than a year ago when merging the initial CCPP version in July 2019.